test/wh_test_crypto.c

```diff
@@ -746,9 +998,12 @@ int whTest_CryptoClientConfig(whClientConfig* config)
     }
     printf("CURVE25519 SUCCESS\n");
 #endif /* HAVE_CURVE25519 */
+#endif
```

Comment:
dead code. Imagine you just want to remove the old code and instead call the new function

Reply:
yup.